pkg/initializer/validator/types.go

```diff
@@ -34,14 +36,16 @@ const (
 	DataDirectoryCorrupt
 	// DataDirectoryStatusUnknown indicates validator failed to check the data directory status.
 	DataDirectoryStatusUnknown
-	// RevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision.
+	// RevisionConsistencyError indicates current etcd revision is inconsistent with latest snapshot revision even after WALs file flushes the data to Bolt DB,if any.
```

Comment:
`even after WALs file flushes the data to Bolt DB,if any` is an implementation detail, and ideally shouldn't be specified in the error description. You can remove this.